src/renderer/src/routes/pages/BuyAndSellTokens/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { useTranslation } from 'react-i18next'
+import { TbShoppingBag } from 'react-icons/tb'
+import { Location, useLocation, useNavigate } from 'react-router-dom'
+import { HelpButton } from '@renderer/components/HelpButton'
+import { useMountUnsafe } from '@renderer/hooks/useMount'
+import { useAppDispatch } from '@renderer/hooks/useRedux'
+import { ContentLayout } from '@renderer/layouts/ContentLayout'
+import { MainLayout } from '@renderer/layouts/Main'
+import { buyAndSellTokensReducerActions } from '@renderer/store/reducers/BuyAndSellTokensReducer'
+import { IAccountState } from '@shared/@types/store'
+
+import { BuyAndSellTokensContainer } from './BuyAndSellTokensContainer'
+import { BuyAndSellTokensProvider } from './context'
+
+type TLocationState = {
+  account?: IAccountState
+}
+
+export const BuyAndSellTokensPage = () => {
+  const { t } = useTranslation('pages', { keyPrefix: 'buyAndSellTokens' })
+  const { state } = useLocation() as Location<TLocationState>
+  const navigate = useNavigate()
+  const dispatch = useAppDispatch()
+  const account = state?.account
+
+  const rightComponent = (
+    <div className="flex gap-x-2">
+      <HelpButton />
+    </div>
+  )
```

Comment:
Create a new component outside this component
```suggestion
  const RightComponent = (
    <div className="flex gap-x-2">
      <HelpButton />
    </div>
  )
```